### DISCUSSION TITLE
Local force closed channels and can't recover some funds

### DISCUSSION THREAD

**Discussion by Silarous** [lightningnetwork/lnd#7941] [Troubleshooting] [answered]
For some context, my original node was a basic Bitcoin/LND setup running on Ubuntu Server.  I decided to try Umbrel and was porting my current LND node over to the Umbrel setup.  I loaded the 24 word seed and then my dumb-ass restored the latest channel.backup, completely forgetting that would close all the channels.

Most of the channels closed successfully (have 1 still stuck due to low fee) and most of the funds did return on-chain.  However, for some reason I have 5 utxo's that are stuck with zero confirmations.  It appeared that these were also due to a low 1sat//vB fee.  I attempted to bump the fee of one of the transactions with 

lncli wallet bumpfee --sat_per_byte 50 TX

That appeared to work and started a CPFP and brought the ancestor TX up to 19sat/vB.  This transaction appeared in my local mempool.space and I thought all was well.  Roughly a week later and the transaction is still unconfirmed.  I checked other, remote block explorers and the transaction is not appearing anywhere other than my own node.  It seems the transaction is not being broadcast.  What am I missing or doing wrong?

**Comment by guggero** [comment]
The problem is that with the current number of unconfirmed transactions, `bitcoind` with its default mempool size of 300MB will start dropping transactions with the lowest fees. So your 1 sat/vByte transactions cannot even make it into your local `bitcoind`'s mempool and hence also cannot be broadcast. You can try setting `maxmempool=1024` in your `bitcoind` config and then try the bump fee again, hoping the 1 sat/vByte transactions will eventually reach a miner.

**Reply by Silarous** [reply]
Unfortunately, my maxmempool is already at 1GB.  The transaction appears to be staying in my own mempool.  You think this will eventually clear if the mempool drops below 300MB again?

**Reply by guggero** [reply]
Ah, okay. I guess what also helps is to update to the latest version of `lnd` (v0.16.4-beta), which re-broadcasts the transactions at every new incoming block to help with those issues. But yes, eventually they will confirm, if the mempool clears.
Another option is to use a transaction accelerator, but those are usually very expensive. But it looks like there's a "free" allowance here? https://www.viabtc.com/tools/txaccelerator/

**Reply by Silarous** [reply]
Thanks.  I am currently on the latest LND.  As much as I'd love to see it, I doubt the mempool will clear anytime soon...  It shouldn't have to completely clear though right?  I thought the CPFP was there to alleviate this issue.  The ancestor's transaction is at an effective 19sat/vB when considering the child.  Fortunately, it's not a large sum of BTC.

**Reply by guggero** [reply, marked as answer]
The mempool just has to clear as much in order for 1 sat/vByte transactions to not be purged anymore. You can see the current "purge" rate on mempool.space. Unfortunately CPFP doesn't really solve this, as you still need to be able to propagate the parent transaction into other mempools. Only package relay (change in Bitcoin Core) will reliably solve this issue.

**Reply by Silarous** [reply]
The stuck UTXO's did finally clear with the bumped fee and are good to go!  Unfortunately, I have one remaining channel closure that is stuck with a low fee.  I tried bumping it multiple times with 

lncli wallet bumpclosefee --sat_per_vbyte 50 

That doesn't seem to bump the fee the same way the stuck UTXOs did.  The transaction remains the same at 1.60 sat/vB.  The transaction is showing in public block explorers.

**Reply by Silarous** [reply]
Fortunately, the mempool cleared enough this past weekend that it finally processed transaction.  After about 2 months of waiting, I'm good to go.
